idaes/power_generation/properties/flue_gas_ideal.py

```diff
@@ -61,191 +75,301 @@ class FlueGasParameterData(PhysicalParameterBlock):
-            doc='Constants for spec. heat capacity for ideal gas',
-            units=pyunits.J/pyunits.mol/pyunits.K)
+            doc="Constants for spec. heat capacity for ideal gas",
+            units=pyunits.J / pyunits.mol / pyunits.K,
+        )
         self.cp_mol_ig_comp_coeff_H = Param(
             self.component_list,
             initialize=cp_mol_ig_comp_coeff_parameter_H,
-            doc='Constants for spec. heat capacity for ideal gas',
-            units=pyunits.kJ/pyunits.mol)
+            doc="Constants for spec. heat capacity for ideal gas",
+            units=pyunits.kJ / pyunits.mol,
+        )
 
         # Viscosity and thermal conductivity parameters
         self.ce_param = Param(
             initialize=2.6693e-5,
-            units=(pyunits.g**0.5*pyunits.mol**0.5*pyunits.angstrom**2 *
-                   pyunits.K**-0.5*pyunits.cm**-1*pyunits.s**-1),
-            doc="Parameter for the Chapman-Enskog viscosity correlation")
+            units=(
+                pyunits.g ** 0.5
+                * pyunits.mol ** 0.5
+                * pyunits.angstrom ** 2
+                * pyunits.K ** -0.5
+                * pyunits.cm ** -1
+                * pyunits.s ** -1
+            ),
+            doc="Parameter for the Chapman-Enskog viscosity correlation",
+        )
 
         self.sigma = Param(
             self.component_list,
             initialize={
-                'O2': 3.458,
-                'N2': 3.621,
-                'NO': 3.47,
-                'CO2': 3.763,
-                'H2O': 2.605,
-                'SO2': 4.29},
-            doc='collision diameter in Angstrom (10e-10 m)',
-            units=pyunits.angstrom
+                k: v
+                for k, v in {
+                    "O2": 3.458,
+                    "N2": 3.621,
+                    "NO": 3.47,
+                    "CO2": 3.763,
+                    "H2O": 2.605,
+                    "SO2": 4.29,
+                }.items()
+                if k in self.component_list
+            },
+            doc="collision diameter in Angstrom (10e-10 m)",
+            units=pyunits.angstrom,
         )
         self.ep_Kappa = Param(
             self.component_list,
             initialize={
-                'O2': 107.4,
-                'N2': 97.53,
-                'NO': 119.0,
-                'CO2': 244.0,
-                'H2O': 572.4,
-                'SO2': 252.0},
+                k: v
+                for k, v in {
+                    "O2": 107.4,
+                    "N2": 97.53,
+                    "NO": 119.0,
+                    "CO2": 244.0,
+                    "H2O": 572.4,
+                    "SO2": 252.0,
+                }.items()
+                if k in self.component_list
+            },
             doc="characteristic energy of interaction between pair of "
-                "molecules, K = Boltzmann constant in Kelvin",
-            units=pyunits.K)
+            "molecules, K = Boltzmann constant in Kelvin",
```

Comment:
Maybe "reduced by the Boltzmann constant to put in units of Kelvin"?

Reply:
I didn't write this, so it took a bit to find the correct description.